awsauthenticationlib/awsauthenticationlib.py

```diff
@@ -162,7 +166,9 @@ def __init__(self, arn):
     def _get_assumed_role(self, arn):
         self.logger.debug('Trying to assume role "%s".', arn)
         try:
-            return self._sts_connection.assume_role(RoleArn=arn, RoleSessionName="AssumeRoleSession")
+            return self._sts_connection.assume_role(RoleArn=arn,
+                                                    RoleSessionName="AssumeRoleSession",
+                                                    DurationSeconds=self.session_duration)
```

Comment:
I think that this would break control tower but don't really remember the details.